sites/docs/content/index.md

```diff
@@ -4,14 +4,16 @@ description: What is this?
 tagline: Guide
 ---
 
-<script>
-	import { Callout } from '$lib/components'
-</script>
-
 Mode Watcher provides simple utilities to manage light & dark mode in your SvelteKit app.
 
-<Callout type="warning">
-
-The documentation is still a work in progress. Please refer to the [GitHub README](https://github.com/svecosystem/mode-watcher) for more information.
+## Features
```

Comment:
for this listing of features, I took some from the landing page and others I came up with myself.

let me know if you would rather have them exactly as they are listed on the landing page!

Reply:
Looks great! Feel free to update the ones on the landing page if you want!